src/handlers/comment/handlers/query.ts

```diff
@@ -0,0 +1,31 @@
+import { getBotContext, getLogger } from "../../../bindings";
+import { Payload } from "../../../types";
+import { getWalletAddress, getWalletMultiplier } from "../../../adapters/supabase";
+
+export const multiplier = async (body: string) => {
+  const context = getBotContext();
+  const logger = getLogger();
+  const payload = context.payload as Payload;
+  const sender = payload.sender.login;
+
+  logger.info(`Received '/query' command from user: ${sender}`);
+
+  const issue = payload.issue;
+  if (!issue) {
+    logger.info(`Skipping '/query' because of no issue instance`);
+    return;
+  }
+
+  const regex = /\S+\s+@(\S+)/;
+  const matches = body.match(regex);
+  const user = matches?.shift()
+  if (user) {
+    const walletAddress = getWalletAddress(user)
+    const multiplier = getWalletMultiplier(user)
```

Comment:
Both wallet address and multiplier are in the `wallets` table so should be able to get both with one request

Reply:
done